charts/pulsar/templates/broker/function-worker-configfile-configmap.yaml

```diff
@@ -84,6 +84,19 @@ data:
       changeConfigMapNamespace: {{ template "pulsar.namespace" . }}
       submittingInsidePod: true
       installUserCodeDependencies: true
+  {{- if .Values.functions.functionMesh }}
+  functionsWorkerServiceNarPackage: "/pulsar/lib/mesh-worker-service.nar"
+  functionsWorkerServiceCustomConfigs:
+    extraDependenciesDir: "/pulsar/lib"
+    functionEnabled: "true"
+    sinkEnabled: "true"
+    sourceEnabled: "true"
+    uploadEnabled: "true"
+    functionRunnerImages:
+      GO: "streamnative/pulsar-functions-go-runner:2.9.2.9"
```

Comment:
Can we make these settings configurable under the `images` section in the values file?